.github/workflows/release.yml

```diff
@@ -12,8 +12,11 @@ jobs:
         uses: actions/checkout@v2
       - name: Build
         run: make release
+        working-directory: Packages/CatbirdApp
       - name: Archive
-        run: zip -r catbird.zip catbird start.sh stop.sh LICENSE Public Resources Sources/CatbirdAPI
+        run: |
+          cd ..
+          zip -r catbird.zip Packages/CatbirdApp/catbird Packages/CatbirdApp/start.sh Packages/CatbirdApp/stop.sh LICENSE Public Resources
```

Comment:
working-directory would simplify the creation of an archive

Reply:
No, because `LICENSE`, `Public` and `Resources` folders are located in the root.

Reply:
I've moved `Public` and `Resources` folders and place a copy of `LICENSE` file to the `Packages/CatbirdApp` folder.